rps.js

```diff
@@ -41,5 +32,18 @@ var compare = function(choice1, choice2){
   }
 }
 
-compare(userChoice, computerChoice);
-});
\ No newline at end of file
+$("#userChoice button").click(function(){
+  userChoice = $(this).val();
```

Comment:
`$(this)` to the particular button that was clicked, so it takes the value of the button (which you can see in the html doc),  and assigns it to userChoice.